mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -98,13 +67,16 @@ class StopWordsRemover(override val uid: String)
 
   /**
    * the stop words set to be filtered out
-   * Default: [[StopWords.English]]
+   * Default: [[StopWords.languageMap("english")]]
```

Comment:
It won't show up correctly since `object StopWords` is private. `StopWords.English` is not correct either. We can just mention that by default it is unset and it will be set to the default list for the selected language during transformation.